src/pipecat/processors/gstreamer/pipeline_source.py

```diff
@@ -0,0 +1,233 @@
+#
+# Copyright (c) 2024, Daily
+#
+# SPDX-License-Identifier: BSD 2-Clause License
+#
+
+import asyncio
+
+from openai import BaseModel
```

Comment:
Why do we import the openai `BaseModel` class instead of directly importing the pydantic `BaseModel` class?

https://github.com/openai/openai-python/blob/main/src/openai/_models.py#L81

Reply:
Oopps, good catch. This should be pydantic. I probably chose the wrong one from automatic import.